components/lib/calendar/Calendar.js

```diff
@@ -3685,6 +3689,28 @@ export const Calendar = React.memo(
             return null;
         };
 
+        const checkDateIsDisabled = (month, year) => {
+            const daysCountInAllMonth = month === -1 ? new Array(12).fill(0).map((_, i) => getDaysCountInMonth(i, year)) : [getDaysCountInMonth(month, year)];
```

Comment:
should use `monthPickerValues()`  replaced

Reply:
Not sure what this means?

Reply:
`new Array(12).fill(0)` ->`monthPickerValues()`